setup/hr_employee_current_contract/.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```

Comment:
I think that we should not commit .eggs ?
So, you can add a line in .gitignore for that